packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { ChainId, getContractAddressesForChainOrThrow } from '@0x/contract-addresses';
 import { FillQuoteTransformerOrderType } from '@0x/protocol-utils';
+// TODO(kimpers): does this belong here?
+import { valueByChainId } from '@0x/token-metadata';
```

Comment:
It's a bit weird to import this from here, but we used to it duplicated here and in `token-metadata`. In that case you would need to add any new chain in both places, which tripped me up initially.

I think it could be better to just move it here for now, rather than duplicating but I'm also open to reverting back to the previous duplication. Wdyt?

Reply:
I'm not entire sure the purpose of `@0x/token-metadata` if we also have addresses declared here as well.

Reply:
I think it was originally created as a way to share the token metadata between 0x API and the data service.

As a follow-up to this, it might make sense to move **all** to token related data into that package and stop having addresses sprinkled everywhere 😂

Reply:
I'm reverting this change in order to remove the unwieldy circular dependency that has come up here